### PR TITLE
Added the new proteus filter to the database seeds file

### PR DESCRIPTION
**Pull request by drewpotter**
Added `ContentType` of "Proteus Technical Briefing Note" to database seeds.

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -8,12 +8,14 @@
...
   [["Poster", "Posters"],
-   ["Tool", "Tools / Applications"],
+   #["Tool", "Tools / Applications"],
    ["Dataset", "Spatial Data / Maps"],
    ["Book", "Books"],
    ["Report", "Reports"],
    ["Journal Paper", "Journal Papers"],
-   ["News", "News"]].each do |type|
+   ["News", "News"],
+   ["Online Tool", "Online Tools"],
```
Comment (Levia) [inline db/seeds.rb]: Was this requested as well?
I didn't see that in the original email

File: db/seeds.rb [hunk 2]
```diff
@@ -8,12 +8,14 @@
...
+   ["Proteus Technical Briefing Note", "Proteus Technical Briefing Notes"]].each do |type|
     ContentType.create(singular: type[0], plural: type[1])
```
Comment (Levia) [inline db/seeds.rb]: Maybe we could improve the indentation here.

File: db/seeds.rb (hunk 1 above)
Comment (drewpotter) [reply db/seeds.rb]: I noticed that the website has `Online Tool` but it is missing from the seeds file!

File: db/seeds.rb (hunk 2 above)
Comment (drewpotter) [reply db/seeds.rb]: Sure thing thanks @Levia

**Pull request merged**